js/components/accessibilityControls.js

```diff
@@ -147,11 +145,30 @@ AccessibilityControls.prototype = {
     }
     this.vrRotationAllowed = !isKeyDown; //prevent repeat of keyDown
     this.controller.moveVrToDirection(false, keyDirectionMap[charCode]); //stop rotation if isKeyDown === false or prevent prev rotation if press a button (isKeyDown === true)
+
+    var inPrevKeyPressedArrIndex = -1;
```

Comment:
It seems like this index and the for loop before is only used if isKeyDown is false in the if statement directly below. If this is the case, can we move this block of code into the if statement?

Reply:
\+